ktor-client/ktor-client-okhttp/jvm/src/io/ktor/client/engine/okhttp/OkHttpWebsocketSession.kt

```diff
@@ -81,6 +89,7 @@ internal class OkHttpWebsocketSession(
     override fun onFailure(webSocket: WebSocket, t: Throwable, response: Response?) {
         super.onFailure(webSocket, t, response)
 
+        if (!originResponse.isCompleted) originResponse.completeExceptionally(t)
```

Comment:
There is no reason to do if-check here since it is safe to invoke `completeExceptionally` twice or after successful completion.